diff -Naur mod_video-0.1.0/mod_video.c mod_video-0.1.0-p/mod_video.c
--- mod_video-0.1.0/mod_video.c	Sat Sep  9 06:02:30 2000
+++ mod_video-0.1.0-p/mod_video.c	Sat Aug 18 17:09:44 2001
@@ -403,7 +403,7 @@
     }
 
     /* select correct video channel */
-    if (select_video_channel(conf->input, device)) {
+    if (conf->input >= 0 && select_video_channel(conf->input, device)) {
 	close(device);
 	return SERVER_ERROR;
     }
